### PR TITLE
Update node-mysql-wrapper - works with the newest 2.0.5 version

### PR DESCRIPTION
**Pull request by ghost**
Node-mysql-wrapper now is written in typescript, some changes were made, new d.ts file also created.

### CODE REVIEW

File: node-mysql-wrapper/node-mysql-wrapper-tests.ts [hunk 1]
```diff
@@ -1,18 +1,24 @@
 ///<reference path='./node-mysql-wrapper.d.ts' />
 
-import wrapper = require("node-mysql-wrapper");
-var db = wrapper("mysql://kataras:pass@127.0.0.1/taglub?debug=false&charset=utf8");
+var wrapper = require("node-mysql-wrapper");
```
Comment (horiuchi) [inline node-mysql-wrapper/node-mysql-wrapper-tests.ts]: This change looks bad to me.
We should use `import` for import library.

Comment (kataras) [inline node-mysql-wrapper/node-mysql-wrapper-tests.ts]: Change it to import if you want, both ways are working I don't know which is the best for simple test file

**Comment by mhegazy** [comment]
@kataras, thanks for your contributions; can you please respond to the comments and update the PR.

**Comment by RyanCavanaugh** [comment]
Hello and thank you for your contribution!

Due to an excessively long queue of pull requests that have become stale, we are "declaring bankruptcy" and closing all PRs opened before May 1, 2016. If you'd still like to merge this code in, please open a new PR that has been merged and rebased with the `master` branch.

Going forward, we are committing to review or merge all PRs on a regular basis so this bankruptcy will not occur again. We apologize for the incovenience and hope you will continue to contribute to DefinitelyTyped in the future.

Thanks
The TypeScript and DefinitelyTyped teams

**Pull request closed without merging**